molconfgen/output.py

```diff
@@ -0,0 +1,79 @@
+# Richard Richardson wrote this ;)
+# Function to add a box to the output of
+# sampler.generate_conformers
+
+import MDAnalysis as mda
+import MDAnalysis.transformations
+import numpy as np
+
+
+# Function for finding the largest radius in
+# the trajectory
+# this will come in handy later
+def largest_r(u):
+    r = 0
+    for ts in u.trajectory:
+        ag = u.atoms
+        rnew = ag.bsphere()[0]
+        if rnew > r:
+            r = rnew
+    return r
+
+def write_pbc_trajectory(u, filename, box = None):
+    ''' u is MDAnalysis universe with a trajectory
+     filename is the name of the trajectory output file
+     box argument describes how to set the box dimensions
+
+     I am not quite sure how to read a lone trajectory file
+     and sampler.generate_conformers returns a universe with
+     a trajectroy so for now I'm going to assume a universe 
+     with a trajectory '''
+
+# first case is for if they want no box
+# just return the boxless trajectory I guess
+    if box == None:
+        u.atoms.write(filename, frames = "all")
+        return filename    
+
+# second case is for if they want a box but 
+# don't define it themselves
+# here we define a "good" box as 3 times the 
+# largest bsphere radius
+    if box == 'auto':
+        # call largest_r to find the largest r
```

Comment:
no need to comment on the obvious: you chose the function name to be explicit so the code speaks for itself